docs/user-guide/cli-configuringcli-ev.md

```diff
@@ -53,23 +56,11 @@ Setting the log level to `TRACE` or `ALL` might result in sensitive data being l
-| Windows (only) | `ZOWE_DAEMON_PIPE` | Lets you override the last two segments of the name of the communication pipe between the daemon executable (.exe) and the daemon. | Any valid path on your computer | `\\.\pipe\%USERNAME%\ZoweDaemon
+By default, the CLI daemon mode binary creates or reuses a file in the user's home directory each time a Zowe CLI command runs. In some cases, this behavior might be undesirable. For example, when the home directory resides on a network drive and has poor file performance.
 
-## Setting other environment variables
+To change the location that the daemon uses, set the environment variables that are described in the following table.
 
 | Platform | Environment variable  | Description | Values | Default |
 | ---------------------- | ---------------------- | ---------------------- | ---------------------- | ---------------------- |
-| All | `ZOWE_V3_ERR_FORMAT` | For Zowe V2, reformats the message displayed in REST request errors so problem details, and service response and diagnostic information, display in a reader friendly manner. In Zowe V3, this will be the only error format used and this environment variable will not be available.| `TRUE`, `FALSE`, blank | blank |
-| All | `CI` | Set by most Continuous Integration environments automatically. Set to any value, disables progress bars in Zowe CLI. | Any <br/>(CI environment name, typically) | blank |
-| All | `FORCE_COLOR` | For most CLI tools, sets the color depth to be used by the CLI on the terminal. Set to `0`, disables color and progress bars in Zowe CLI. Set to any other valid, non-blank value, enables color and progress bars in Zowe CLI. <br/><br/> See the subsequent Note regarding Zowe CLI daemon configuration. | `0`, `1`, `2`, `3`, `TRUE`, blank | blank |
-
-:::note
-
-When a user does not set `FORCE_COLOR` and uses the Zowe CLI daemon, the daemon determines if the terminal running the daemon supports colors and progress bars. If it does, the daemon automatically sets `FORCE_COLOR` to a supported setting in all requests sent to the Zowe CLI daemon server component.
-
-:::
+| All | `ZOWE_DAEMON_DIR` | Lets you override the complete path to the directory that will hold daemon files related to this user. The directory can contain the following files:<ul><li>`daemon.lock`</li><li>`daemon.sock`</li><li>`daemon_pid.json`</li></ul> | Any valid path on your computer | `<your_home_dir>/.zowe/daemon` <br/><br/> Examples: <br/><br/> Windows: `%HOMEPATH%\.zowe\daemon`<br/>Linux: `$HOME/.zowe/daemon` |
+| Windows (only) | `ZOWE_DAEMON_PIPE` | Lets you override the last two segments of the name of the communication pipe between the daemon executable (.exe) and the daemon. | Any valid path on your computer | `\\.\pipe\%USERNAME%\ZoweDaemon` **[correct?]**
```

Comment:
@adam-wolfe @zFernand0 - is this correct? The original version of this file had this: **`\.\pipe%USERNAME%\ZoweDaemon**
The fact that there was only one backtick confused me, so I added a closing backtick and the characters in the path changed slightly. Please advise.

Reply:
There should definitely be a `\` after pipe. Regarding "Any valid path on your computer", it might be a bit confusing to use the same wording as the ZOWE_DAEMON_DIR env var which points to an actual directory path. Would it make more sense to say "Any valid pipe name on your computer", or would that be more confusing and/or wrong?

Reply:
Yes, it is accurate, thanks for fixing this!

Here is the source code that shows what it should does on the code. 
https://github.com/zowe/zowe-cli/blob/79f5e8c3f356d4144bb6365acf4346c82a5da9c4/packages/cli/src/daemon/DaemonDecider.ts#L203

No need to duplicate every backslash (as it is in the code)

This displays just fine
![image](https://github.com/zowe/docs-site/assets/37381190/cc6f0d35-e75d-41d1-9b2d-c7915e7ad5be)